Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM omegacen/conda-builder
+
+LABEL maintainer="H.Buddelmeijer@rug.nl"
+LABEL version="1.0"
+LABEL description="Image for building LaTeX documentation through tectonic"
+
+ARG CONDA_PREFIX=/opt/conda
```

Comment:
I'm a little bit foggy on this -- is this not inherited from the parent image / Dockerfile?

Reply:
Apparently not (because it didn't work without this declaration). It makes sense though, because there is no link to the parent Dockerfile, because there is no link between images and their Dockerfile. So there is no way for the child Dockerfile to know what information is in the parent Dockerfile.

(There not being any link between the Dockerfile and the image makes me a bit uncomfortable though. It feels like a house of cards. E.g. I had to look in our registry to figure out what the name of the base image was.)

Or would there be a way to have another Dockerfile as the `FROM` instruction?